src/config/mod.rs

```diff
@@ -20,6 +20,8 @@ use tracing::{error, info, warn};
 mod parse;
 pub mod types;
 
+#[derive(Debug)]
+
```

Comment:
NIT Can you remove this extra line break? Also if you update the changelog with this change, then it looks good to go :+1:

Reply:
Sorry about that one :facepalm:, I'll fix it right away.